.travis.yml

```diff
@@ -71,11 +87,11 @@ jobs:
     script: ./scripts/build_posix.sh
   - name: "PyPI pkgs"
     language: python
-    python: 3.7
+    python: 3.8
     script: ./scripts/build_package.sh
   - name: Snapcraft snap
     language: python
-    python: 3.7
+    python: 3.8
```

Comment:
Are we confident enough to start *building* using 3.8?

Reply:
Good point. We can keep Snap on `3.7` and wait on feedbacks of Python packages? What do you think?

Reply:
I'd say the other way around if we want to field test. `snap` packages are super easy for us to auto roll-back in case of issues. We don't have as much control over broken PyPI releases.

Reply:
@efiop not sure if your :+1: means "confident" or "hesitant."

Reply:
It is a good question. Looks like the packages are indeed getting built successfully, so I would be fine trying it out with 3.8. Pypi packages are for all versions, since we don't have any c extensions in dvc, so I'm not too worried about them.